Gems/PhysX/Code/Source/ArticulationLinkComponent.cpp

```diff
@@ -271,7 +271,7 @@ namespace PhysX
 
     void setInboundJointDriveParams(
         physx::PxArticulationJointReducedCoordinate* inboundJoint,
-        physx::PxArticulationAxis articulationAxis,
+        [[maybe_unused]] physx::PxArticulationAxis articulationAxis,
```

Comment:
Shouldn't this get passed to the `setDriveParams` function below?

Reply:
I don't know but I can look into it. I'm surprised others are not seeing this failure.

Reply:
I see. It is unusual that it's hard coded to twist. I'm not sure why it's not passing in that parameter. Let's mark it as maybe unused for now to unblock anyone else who's having issues building. I'm not familiar enough with it to  Change to pass in the variable and validated before the cutoff.

Reply:
That is good with me.